AsyncDisplayKit/ASNetworkImageNode.mm

```diff
@@ -38,17 +38,19 @@ @interface ASNetworkImageNode ()
   id _downloadIdentifier;
 
   BOOL _imageLoaded;
-  
+  CGFloat _currentImageQuality;
```

Comment:
Could you add a block comment, maybe here or in the displayDidFinish, describing the overall management strategy - particularly the subtleties around the thread trampoline?